docs/setup/independent/high-availability.md

```diff
@@ -0,0 +1,328 @@
+# High Availability
+
+## Set up etcd
+
+1. Create 3 Virtual Machines that follow [CoreOS' hardware recommendations](https://coreos.com/etcd/docs/latest/op-guide/hardware.html).
```

Comment:
What about quoting possible alternatives to an external ectd cluster?

Reply:
What alternatives do you have in mind? IMO there no HA alternatives

Reply:
sorry, I wasn't clear in providing my feedback.
My proposal here is to list other possible way to deploy an HA etcd cluster, each one with caveats/limitation with respect to the proposed solution: 
- running etcd in docker, as suggested by @luxas 
- running etcd/systemd (or docker) on master nodes (thus avoiding the costs of three VM)
- running etcd in kubernetes (thus unifying - at least in part - etcd management with management of other workloads)

Reply:
I too would be interested what makes three VMs preferrable to three master nodes for the etcd cluster?
To me that solution seems just fine.